classes/person.rb

```diff
@@ -3,18 +3,21 @@
 require_relative 'capitalize_decorator'
 require_relative 'trimmer_decorator'
 require_relative 'base_decorator'
+require_relative 'rental'
+require_relative 'book'
 
 class Person < Nameable
   def initialize(age, name, parent_permission: true)
     @id = rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
 
   attr_reader :id
```

Comment:
- [ ] Make sure that you create a method for adding rentals in the Person class. The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Check the example below:
```ruby
def add_rental(book, date)
    Rental.new(date, self, book)
end
```